contents/blog/best-hipaa-compliant-ab-testing-tools.md

```diff
@@ -0,0 +1,129 @@
+
+Within the Privacy Rule framework, there are three different ways to comply with HIPAA when adopting user analytics and experimentation tools:
+
+1. **Anonymize all PHI and identifiers:** This is an obvious starting point. There are two so-called '[De-indentification Standards](https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#standard)' – 'Expert Determination', where an expert verifies that data isn't personally identifiable; and 'Safe Harbor' where all 18 types of identifier are removed. The former is preferable simply because applying the Safe Harbor approach can render data effectively useless for analytical purposes.
+
+2. **Sign a BAA with a third-party tool:** You must sign a BAA with any third-party platform that handles your protected health information (PHI). This can mean signing multiple agreements, though, such as one with your analytical partner, but also any tools you use for importing and exporting data from your data warehouse.
+
+3. **Self-host and keep control of all your data:** The less common, but arguably most powerful option, is to self-host tools for analytics and experimentation on your own infrastructure. This reduces the number of BAA's and general legal shenanigans needed to generate user insights. The only downside is you'll need the expertise to manage self-hosted instances, or third-party support to do so.
+
+These are the broad principles, but **please consult an expert** before making any final decision on how to implement tools in compliance with HIPAA. 
+
+## The best HIPAA-compliant A/B testing tools
+
+
+
+### PostHog
+
+![posthog experimentation](../images/blog/hipaa-compliant-ab-testing/posthog.png)
+
+PostHog is an all-in-one platform that includes a [fully-featured experimentation](https://posthog.com/product/experimentation-suite) suite in addition to product analytics, session recording, and a feature flag system that also powers its testing features. It supports both A/B/n and multivariate experiments, which you can target by geography, cohorts, and user properties.
+
+PostHog is open source, which means you can inspect the code yourself and extend the product via custom integrations or apps to satisfy specific edge-case requirements. As an [all-in-one platform](https://posthog.com/product), PostHog makes it easy to create cohorts of users based on user properties, then run targeted experiments on those users alone using feature flags.
+
+#### PostHog and HIPAA compliance
+
+- **Self-Hosting:** Yes
+- **BAA Available**: No
+
+PostHog doesn't currently offer a BAA PostHog Cloud, so using it in compliance with HIPAA means [self-hosting it](/docs/self-host) on your own infrastructure. The [PostHog Marketplace](/marketplace) includes several third-party providers who can support self-hosted deployment, or manage your instance entirely, if you need help.
```

Comment:
```suggestion
PostHog doesn't currently offer a BAA for using PostHog Cloud, so using it in compliance with HIPAA means [self-hosting it](/docs/self-host) on your own infrastructure. The [PostHog Marketplace](/marketplace) includes several third-party providers who can support self-hosted deployment, or manage your instance entirely, if you need help.
```

Reply:
Technically true - although this may change in the next few weeks as we figure out our own BAA - could we mention that Cloud might be coming soon?

Reply:
Yeah, we could do something similar to what we did on the GDPR one, potentially.

Reply:
In the interests of time, I'm going to go live as is and we can update when needed. I don't expect this to pull hue numbers anyway.